roles/load-ycsb/tasks/main.yml

```diff
@@ -46,7 +46,7 @@
                 value: "{{uuid}}"
               imagePullPolicy: Always
               command: ["/bin/sh"]
-              args: ["-c", "/opt/snafu/ycsb-wrapper/ycsb-wrapper.py -r 1 -l -d {{ ycsb.driver }} -w {{ ycsb_workload }} -x \"{{ ycsb.options_load }}\""]
+              args: ["-c", "/opt/snafu/ycsb_wrapper/ycsb_wrapper.py -r 1 -l -d {{ ycsb.driver }} -w {{ ycsb_workload }} -x \"{{ ycsb.options_load }}\""]
```

Comment:
this line should be `args: ["python /opt/snafu/run_snafu.py --tool ycsb -r 1 -l -d {{ ycsb.driver }} -w {{ ycsb_workload }} -x \"{{ ycsb.options_load }}\""]`